src/PaperG/ScenarioBlob/Blobs/UnmanagedFacebook/FacebookAdSet.php

```diff
@@ -166,4 +166,4 @@ private function validateGoal($validOptimizationGoals)
 
         return in_array($this->optimizationGoal, $validOptimizationGoals);
     }
-} 
+}
```

Comment:
My thought here is that we want to move everything into their respective scenarios, and FacebookAdSet is one where I see it should belong to Facebook scenario. I can move the few FB classes that have been there back to where they belong and we can contemplate where to move them in a future PR. Sounds good?